doc/nrf/ug_matter_gs_testing_thread_separate_otbr_linux_macos.rst

```diff
@@ -39,19 +38,42 @@ To use this setup, you need the following hardware:
 * 1x compatible Nordic Semiconductor's DK - for the Matter accessory device (compatible and programmed with one of :ref:`matter_samples`)
 
 Configuring the environment
-===========================
+***************************
 
-To configure and use Thread Border Router and CHIP Tool for Linux or macOS on separate devices, complete the following steps:
+To configure and use Thread Border Router and CHIP Tool for Linux or macOS on separate devices, complete the following steps.
 
-1. Program the development kit for the Matter accessory device with one of available :ref:`matter_samples`.
-#. Configure the Thread Border Router on a Raspberry Pi.
-   See :ref:`ug_thread_tools_tbr` in the |NCS| documentation for details.
-#. Configure the CHIP Tool for Linux or macOS by following the steps in the sections of the :doc:`matter:chip_tool_guide` in the Matter documentation:
+.. rst-class:: numbered-step
 
-   a. Build and run the CHIP Tool by completing the steps listed in "Building and running CHIP Tool".
-   #. Prepare the environment for testing by completing the steps listed in "Using CHIP Tool for Matter device testing".
+Program the sample
+==================
+
+Program the development kit for the Matter accessory device with one of available :ref:`matter_samples`.
+
+.. rst-class:: numbered-step
+
+Configure the Thread Border Router
+==================================
+
+Configure the Thread Border Router on a Raspberry Pi.
+For detailed steps, see the :ref:`ug_thread_tools_tbr` page in the |NCS| documentation.
+
+.. rst-class:: numbered-step
+
+Configure the CHIP Tool for Linux or macOS
```

Comment:
Shouldn't it be aligned with "Configure the Matter controller" section in the page about one-device solution? That is, contain info about pre-build packages?

Reply:
Strangely enough, the original section did not have this split.

Reply:
Done